src/main/resources/edu/wpi/first/outlineviewer/MainWindow.fxml

```diff
@@ -19,6 +19,8 @@
         <MenuBar BorderPane.alignment="CENTER">
             <Menu mnemonicParsing="false" text="File" id="fileMenu">
                 <MenuItem onAction="#showPreferences" text="Preferences" id="preferencesMenuButton" accelerator="SHORTCUT+,"/>
+                <MenuItem onAction="#loadState" text="Load state" id="loadStateButton" accelerator="SHORTCUT+O"/>
+                <MenuItem onAction="#saveState" text="Save state" id="saveStateButton" accelerator="SHORTCUT+S"/>
```

Comment:
I think I would order it like:
>load
save
SeparatorMenuItem
preferences
exit

Thoughts?